docs/Keepers/introduction.md

```diff
@@ -2,28 +2,54 @@
 layout: nodes.liquid
 section: smartContract
 date: Last Modified
-title: 'Introduction to Chainlink Keepers (Beta)'
+title: 'Introduction to Chainlink Keepers'
 permalink: 'docs/chainlink-keepers/introduction/'
 whatsnext:
   {
     'Making Keeper Compatible Contracts': '/docs/chainlink-keepers/compatible-contracts/',
   }
 ---
+
 ![Chainlink Keeper Network](/images/contract-devs/generic-banner.png)
 
 {% include keepers-beta %}
 
 A major limitation of smart contracts is that they can't trigger or initiate their own functions at arbitrary times or under arbitrary conditions. State change will only occur when a transaction is initiated by another account (such as user, oracle, or contract).
 
-Chainlink Keepers allow you to register Upkeep for your contract. When the conditions you specify are met, the Chainlink Keeper Network will execute a method on your contract. [Learn how the network works.](../overview)
+To solve this problem, blockchain projects can:
+
+1. Create highly reliable infrastructure trigger smart contract functions - centralized and expensive to build/maintain,
+1. Outsource it to a 3rd party - possibly centralized or having a single point of failure in an external party, or with complex incentive alignment
+1. Use the Chainlink Keeper Network
```

Comment:
The changes here miss the target a bit. I've been talking to a number of projects recently and I think the distinction between 3rd party outsourcing and open systems are two independent concepts that are well recognized. The open market solutions have reliability concerns hence why Chainlink Keepers is attractive.

2. Outsource it to a 3rd party - often centralized and a single point of failure in an external party
3. Use an open market solution - decentralized, but comes along with complex incentive alignment, potential for competitive bots that increase the cost of execution, and ensuring reliability can be difficult
4. Use Chainlink Keeper Network

Reply:
updated with
```
1. Create highly reliable infrastructure trigger smart contract functions - centralized and expensive to build/maintain,
1. Outsource it to a 3rd party - often centralized and a single point of failure
1. Use an open market solution - decentralized, but can come with complex incentive alignment, the potential for competitive bots that increase the cost of execution, and difficulties ensuring reliability
1. Use the Chainlink Keeper Network```